.github/workflows/native-unix.yml

```diff
@@ -169,22 +169,30 @@ jobs:
           mamba install -c conda-forge \
             --file ci/conda_env_cpp.txt
 
+      - name: Export ASAN/UBSAN environment variables
+        shell: bash -l {0}
+        run: |
+          echo "ASAN_OPTIONS=detect_leaks=0:symbolize=1:strict_init_order=true:allocator_may_return_null=1:halt_on_error=1" >> "$GITHUB_ENV"
```

Comment:
I think we don't need to copy NumPy's config exactly. We can remove `detect_leaks=0` to keep LeakSanitizer enabled. The other flags seem fine enough.

Reply:
Sounds good. FYI when using this in coordination with Python if you don't ignore leaks you get a ton of (false?) positives from the CPython runtime.

Reply:
Ah, ok, then we should keep it - thanks for the explanation

Reply:
Ah, ok, then we should keep it - thanks for the explanation